File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.PersonService.Models/get-person_item.ts


import { IGetPerson_Item } from './iget-person_item';

export class GetPerson_Item implements IGetPerson_Item {
  
	public PersonID: number = 0;
	public CountryID: number = 0;
	public FullName: string = "";
	public Gender: string = "";
	public AgencyName: string = "";
	public RankName: string = "";
	public JobTitle: string = "";
	public IsUSCitizen?: boolean;
	public NationalID: string = "";
	public ContactEmail: string = "";
	public ContactPhone: string = "";
	public DOB?: Date;
	public PlaceOfBirth: string = "";
	public PlaceOfResidence: string = "";
	public EducationLevel: string = "";
	public PersonLanguagesJSON: string = "";
  
}

File: GTTS/INL.PersonService.Models/GetPerson_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.PersonService.Models
{
    public class GetPerson_Item : IGetPerson_Item
    {
        public long PersonID { get; set; }
        public int CountryID { get; set; }
        public string FullName { get; set; }
        public string Gender { get; set; }
        public string AgencyName { get; set; }
        public string RankName { get; set; }
        public string JobTitle { get; set; }
        public bool? IsUSCitizen { get; set; }
        public string NationalID { get; set; }
        public string ContactEmail { get; set; }
        public string ContactPhone { get; set; }
        public DateTime? DOB { get; set; }
        public string PlaceOfBirth { get; set; }
        public string PlaceOfResidence { get; set; }
        public string EducationLevel { get; set; }
        public string PersonLanguagesJSON { get; set; }
    }
}
